requirements/pip.txt

```diff
@@ -4,13 +4,13 @@
 #
 #    make upgrade
 #
-wheel==0.44.0
+wheel==0.45.0
     # via -r requirements/pip.in
 
 # The following packages are considered to be unsafe in a requirements file:
 pip==24.2
     # via
-    #   -c requirements/common_constraints.txt
+    #   -c /home/runner/work/completion/completion/requirements/common_constraints.txt
```

Comment:
FYI this is a known issue with pip-compile: https://github.com/jazzband/pip-tools/issues/2131